tests/test_validators.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from frontstage.common.validators import remove_required, InputRequired, DataRequired
+
+
+class TestRemoveRequiredFunction(unittest.TestCase):
+    """
+    remove_required function tests
+    """
+
+    def test_should_remove_a_required_member_if_there_is_one(self):
+        test_tuple = ('required', 'other', 'somethingelse')
+        output = remove_required(test_tuple)
+
+        self.assertFalse('required' in output)
+
+    def test_should_remove_a_required_if_its_the_only_item_in_list(self):
+        test_tuple = ('required')
+        output = remove_required(test_tuple)
+
+        self.assertFalse('required' in output)
+
+    def test_should_remove_a_required_member_if_its_in_the_middle_of_the_list(self):
+        test_tuple = ('otheritem', 'required', 'other', 'somethingelse')
+        output = remove_required(test_tuple)
+
+        self.assertFalse('required' in output)
+    
```

Comment:
Theres whitespace on this line which is causing linting error

Reply:
Resolved.